modules/installation-azure-service-principal.adoc

```diff
@@ -192,6 +192,19 @@ control. For more information, see https://aka.ms/azadsp-cli
 }
 ----
 
+[NOTE]
+====
+If you want to use custom roles with all the required permissions, you must skip the step 6. You can create the service principal for your account by running the following command:
+[source,terminal]
+----
+az ad sp create-for-rbac --role <custom-role> <1> --name <service_principal> <2> \
+  --scopes /subscriptions/<subscription_id> <3>
+----
+<1> Specify the custom-role name.
+<2> Specify the service principal name.
+<3> Specify the subscription ID.
+====
```

Comment:
In per-reviewing doc, it looks like the <1>/<2> have different format with <3>.

Reply:
Addressed

Reply:
I think it's still not updated @xenolinux , and infact the next step numbers have been changed. 5 has become 1, 6 -> 2

Reply:
checking....

Reply:
The note has been moved to the end of the procedure. 
- Addressed

Reply:
> In per-reviewing doc, it looks like the <1>/<2> have different format with <3>.
I'm still facing this issue.

Reply:
I am looking into it.

Reply:
The format is fixed now